apps/cms/.env.example

```
@@ -0,0 +1,31 @@
+PAYLOAD_ADMIN_EMAIL=dev@comfy.local
+PAYLOAD_ADMIN_PASSWORD=changeme
+
+# Media object storage (GCS). Leave GCS_BUCKET unset for local dev to keep media
+# on the Next server's local disk (/api/media/file/...). Set all of these to
+# serve media from the bucket behind the media.comfy.org CDN — uploads then land
+# in the bucket and each media doc's url becomes an absolute CDN url.
+# GCS_CREDENTIALS_JSON is a service-account key (object-admin on the bucket) as a
+# single-line JSON string — keep it in .env / Vercel only, never commit it.
+GCS_BUCKET=comfy-org-videos
+GCS_PROJECT_ID=dreamboothy
+GCS_CREDENTIALS_JSON={"type":"service_account","project_id":"..."}
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)